[2024-02-27 09:18:47,639] {processor.py:162} INFO - Started process (PID=4618) to work on /opt/airflow/dags/dagrundag1.py
[2024-02-27 09:18:47,641] {processor.py:613} INFO - Processing file /opt/airflow/dags/dagrundag1.py for tasks to queue
[2024-02-27 09:18:47,642] {logging_mixin.py:109} INFO - [2024-02-27 09:18:47,642] {dagbag.py:496} INFO - Filling up the DagBag from /opt/airflow/dags/dagrundag1.py
[2024-02-27 09:18:47,652] {processor.py:623} INFO - DAG(s) dict_keys(['dagrundag2']) retrieved from /opt/airflow/dags/dagrundag1.py
[2024-02-27 09:18:47,950] {logging_mixin.py:109} INFO - [2024-02-27 09:18:47,950] {dag.py:1834} INFO - Sync 1 DAGs
[2024-02-27 09:18:47,962] {logging_mixin.py:109} INFO - [2024-02-27 09:18:47,962] {dag.py:2287} INFO - Setting next_dagrun for dagrundag2 to None
[2024-02-27 09:18:47,979] {processor.py:170} INFO - Processing /opt/airflow/dags/dagrundag1.py took 0.343 seconds
[2024-02-27 09:18:51,144] {processor.py:162} INFO - Started process (PID=4628) to work on /opt/airflow/dags/dagrundag1.py
[2024-02-27 09:18:51,146] {processor.py:613} INFO - Processing file /opt/airflow/dags/dagrundag1.py for tasks to queue
[2024-02-27 09:18:51,147] {logging_mixin.py:109} INFO - [2024-02-27 09:18:51,146] {dagbag.py:496} INFO - Filling up the DagBag from /opt/airflow/dags/dagrundag1.py
[2024-02-27 09:18:51,158] {processor.py:623} INFO - DAG(s) dict_keys(['dagrundag1']) retrieved from /opt/airflow/dags/dagrundag1.py
[2024-02-27 09:18:53,599] {logging_mixin.py:109} INFO - [2024-02-27 09:18:53,597] {dag.py:1834} INFO - Sync 1 DAGs
[2024-02-27 09:18:53,614] {logging_mixin.py:109} INFO - [2024-02-27 09:18:53,613] {dag.py:1853} INFO - Creating ORM DAG for dagrundag1
[2024-02-27 09:18:53,618] {logging_mixin.py:109} INFO - [2024-02-27 09:18:53,618] {dag.py:2287} INFO - Setting next_dagrun for dagrundag1 to None
[2024-02-27 09:18:53,630] {processor.py:170} INFO - Processing /opt/airflow/dags/dagrundag1.py took 2.496 seconds
[2024-02-27 09:18:58,820] {processor.py:162} INFO - Started process (PID=4648) to work on /opt/airflow/dags/dagrundag1.py
[2024-02-27 09:18:58,821] {processor.py:613} INFO - Processing file /opt/airflow/dags/dagrundag1.py for tasks to queue
[2024-02-27 09:18:58,821] {logging_mixin.py:109} INFO - [2024-02-27 09:18:58,821] {dagbag.py:496} INFO - Filling up the DagBag from /opt/airflow/dags/dagrundag1.py
[2024-02-27 09:18:58,828] {processor.py:623} INFO - DAG(s) dict_keys(['dagrundag1']) retrieved from /opt/airflow/dags/dagrundag1.py
[2024-02-27 09:18:58,850] {logging_mixin.py:109} INFO - [2024-02-27 09:18:58,849] {dag.py:1834} INFO - Sync 1 DAGs
[2024-02-27 09:18:58,865] {logging_mixin.py:109} INFO - [2024-02-27 09:18:58,865] {dag.py:2287} INFO - Setting next_dagrun for dagrundag1 to None
[2024-02-27 09:18:58,877] {processor.py:170} INFO - Processing /opt/airflow/dags/dagrundag1.py took 0.061 seconds
[2024-02-27 09:19:04,387] {processor.py:162} INFO - Started process (PID=4659) to work on /opt/airflow/dags/dagrundag1.py
[2024-02-27 09:19:04,389] {processor.py:613} INFO - Processing file /opt/airflow/dags/dagrundag1.py for tasks to queue
[2024-02-27 09:19:04,390] {logging_mixin.py:109} INFO - [2024-02-27 09:19:04,389] {dagbag.py:496} INFO - Filling up the DagBag from /opt/airflow/dags/dagrundag1.py
[2024-02-27 09:19:04,401] {processor.py:623} INFO - DAG(s) dict_keys(['dagrundag1']) retrieved from /opt/airflow/dags/dagrundag1.py
[2024-02-27 09:19:04,424] {logging_mixin.py:109} INFO - [2024-02-27 09:19:04,424] {dag.py:1834} INFO - Sync 1 DAGs
[2024-02-27 09:19:04,436] {logging_mixin.py:109} INFO - [2024-02-27 09:19:04,436] {dag.py:2287} INFO - Setting next_dagrun for dagrundag1 to None
[2024-02-27 09:19:04,449] {processor.py:170} INFO - Processing /opt/airflow/dags/dagrundag1.py took 0.065 seconds
[2024-02-27 09:19:09,023] {processor.py:162} INFO - Started process (PID=4670) to work on /opt/airflow/dags/dagrundag1.py
[2024-02-27 09:19:09,025] {processor.py:613} INFO - Processing file /opt/airflow/dags/dagrundag1.py for tasks to queue
[2024-02-27 09:19:09,026] {logging_mixin.py:109} INFO - [2024-02-27 09:19:09,025] {dagbag.py:496} INFO - Filling up the DagBag from /opt/airflow/dags/dagrundag1.py
[2024-02-27 09:19:09,036] {processor.py:623} INFO - DAG(s) dict_keys(['dagrundag1']) retrieved from /opt/airflow/dags/dagrundag1.py
[2024-02-27 09:19:09,060] {logging_mixin.py:109} INFO - [2024-02-27 09:19:09,060] {dag.py:1834} INFO - Sync 1 DAGs
[2024-02-27 09:19:09,073] {logging_mixin.py:109} INFO - [2024-02-27 09:19:09,073] {dag.py:2287} INFO - Setting next_dagrun for dagrundag1 to None
[2024-02-27 09:19:09,085] {processor.py:170} INFO - Processing /opt/airflow/dags/dagrundag1.py took 0.065 seconds
[2024-02-27 09:19:11,079] {processor.py:162} INFO - Started process (PID=4671) to work on /opt/airflow/dags/dagrundag1.py
[2024-02-27 09:19:11,081] {processor.py:613} INFO - Processing file /opt/airflow/dags/dagrundag1.py for tasks to queue
[2024-02-27 09:19:11,082] {logging_mixin.py:109} INFO - [2024-02-27 09:19:11,082] {dagbag.py:496} INFO - Filling up the DagBag from /opt/airflow/dags/dagrundag1.py
[2024-02-27 09:19:11,098] {processor.py:623} INFO - DAG(s) dict_keys(['dagrundag1']) retrieved from /opt/airflow/dags/dagrundag1.py
[2024-02-27 09:19:11,130] {logging_mixin.py:109} INFO - [2024-02-27 09:19:11,130] {dag.py:1834} INFO - Sync 1 DAGs
[2024-02-27 09:19:11,148] {logging_mixin.py:109} INFO - [2024-02-27 09:19:11,148] {dag.py:2287} INFO - Setting next_dagrun for dagrundag1 to None
[2024-02-27 09:19:11,169] {processor.py:170} INFO - Processing /opt/airflow/dags/dagrundag1.py took 0.093 seconds
[2024-02-27 09:19:13,583] {processor.py:162} INFO - Started process (PID=4681) to work on /opt/airflow/dags/dagrundag1.py
[2024-02-27 09:19:13,586] {processor.py:613} INFO - Processing file /opt/airflow/dags/dagrundag1.py for tasks to queue
[2024-02-27 09:19:13,587] {logging_mixin.py:109} INFO - [2024-02-27 09:19:13,587] {dagbag.py:496} INFO - Filling up the DagBag from /opt/airflow/dags/dagrundag1.py
[2024-02-27 09:19:13,597] {processor.py:623} INFO - DAG(s) dict_keys(['dagrundag1']) retrieved from /opt/airflow/dags/dagrundag1.py
[2024-02-27 09:19:13,625] {logging_mixin.py:109} INFO - [2024-02-27 09:19:13,625] {dag.py:1834} INFO - Sync 1 DAGs
[2024-02-27 09:19:13,640] {logging_mixin.py:109} INFO - [2024-02-27 09:19:13,640] {dag.py:2287} INFO - Setting next_dagrun for dagrundag1 to None
[2024-02-27 09:19:13,654] {processor.py:170} INFO - Processing /opt/airflow/dags/dagrundag1.py took 0.076 seconds
[2024-02-27 09:19:14,676] {processor.py:162} INFO - Started process (PID=4683) to work on /opt/airflow/dags/dagrundag1.py
[2024-02-27 09:19:14,678] {processor.py:613} INFO - Processing file /opt/airflow/dags/dagrundag1.py for tasks to queue
[2024-02-27 09:19:14,679] {logging_mixin.py:109} INFO - [2024-02-27 09:19:14,678] {dagbag.py:496} INFO - Filling up the DagBag from /opt/airflow/dags/dagrundag1.py
[2024-02-27 09:19:14,686] {processor.py:623} INFO - DAG(s) dict_keys(['dagrundag1']) retrieved from /opt/airflow/dags/dagrundag1.py
[2024-02-27 09:19:14,713] {logging_mixin.py:109} INFO - [2024-02-27 09:19:14,713] {dag.py:1834} INFO - Sync 1 DAGs
[2024-02-27 09:19:14,727] {logging_mixin.py:109} INFO - [2024-02-27 09:19:14,727] {dag.py:2287} INFO - Setting next_dagrun for dagrundag1 to None
[2024-02-27 09:19:14,742] {processor.py:170} INFO - Processing /opt/airflow/dags/dagrundag1.py took 0.070 seconds
[2024-02-27 09:19:17,691] {processor.py:162} INFO - Started process (PID=4684) to work on /opt/airflow/dags/dagrundag1.py
[2024-02-27 09:19:17,692] {processor.py:613} INFO - Processing file /opt/airflow/dags/dagrundag1.py for tasks to queue
[2024-02-27 09:19:17,693] {logging_mixin.py:109} INFO - [2024-02-27 09:19:17,693] {dagbag.py:496} INFO - Filling up the DagBag from /opt/airflow/dags/dagrundag1.py
[2024-02-27 09:19:17,705] {processor.py:623} INFO - DAG(s) dict_keys(['dagrundag1']) retrieved from /opt/airflow/dags/dagrundag1.py
[2024-02-27 09:19:17,727] {logging_mixin.py:109} INFO - [2024-02-27 09:19:17,727] {dag.py:1834} INFO - Sync 1 DAGs
[2024-02-27 09:19:17,741] {logging_mixin.py:109} INFO - [2024-02-27 09:19:17,741] {dag.py:2287} INFO - Setting next_dagrun for dagrundag1 to None
[2024-02-27 09:19:17,755] {processor.py:170} INFO - Processing /opt/airflow/dags/dagrundag1.py took 0.069 seconds
[2024-02-27 09:19:23,245] {processor.py:162} INFO - Started process (PID=4695) to work on /opt/airflow/dags/dagrundag1.py
[2024-02-27 09:19:23,247] {processor.py:613} INFO - Processing file /opt/airflow/dags/dagrundag1.py for tasks to queue
[2024-02-27 09:19:23,248] {logging_mixin.py:109} INFO - [2024-02-27 09:19:23,248] {dagbag.py:496} INFO - Filling up the DagBag from /opt/airflow/dags/dagrundag1.py
[2024-02-27 09:19:23,259] {processor.py:623} INFO - DAG(s) dict_keys(['dagrundag1']) retrieved from /opt/airflow/dags/dagrundag1.py
[2024-02-27 09:19:23,588] {logging_mixin.py:109} INFO - [2024-02-27 09:19:23,588] {dag.py:1834} INFO - Sync 1 DAGs
[2024-02-27 09:19:23,605] {logging_mixin.py:109} INFO - [2024-02-27 09:19:23,604] {dag.py:2287} INFO - Setting next_dagrun for dagrundag1 to None
[2024-02-27 09:19:23,624] {processor.py:170} INFO - Processing /opt/airflow/dags/dagrundag1.py took 0.384 seconds
[2024-02-27 09:19:28,938] {processor.py:162} INFO - Started process (PID=4706) to work on /opt/airflow/dags/dagrundag1.py
[2024-02-27 09:19:28,941] {processor.py:613} INFO - Processing file /opt/airflow/dags/dagrundag1.py for tasks to queue
[2024-02-27 09:19:28,943] {logging_mixin.py:109} INFO - [2024-02-27 09:19:28,942] {dagbag.py:496} INFO - Filling up the DagBag from /opt/airflow/dags/dagrundag1.py
[2024-02-27 09:19:28,956] {processor.py:623} INFO - DAG(s) dict_keys(['dagrundag1']) retrieved from /opt/airflow/dags/dagrundag1.py
[2024-02-27 09:19:28,985] {logging_mixin.py:109} INFO - [2024-02-27 09:19:28,984] {dag.py:1834} INFO - Sync 1 DAGs
[2024-02-27 09:19:29,002] {logging_mixin.py:109} INFO - [2024-02-27 09:19:29,002] {dag.py:2287} INFO - Setting next_dagrun for dagrundag1 to None
[2024-02-27 09:19:29,020] {processor.py:170} INFO - Processing /opt/airflow/dags/dagrundag1.py took 0.085 seconds
[2024-02-27 09:19:34,467] {processor.py:162} INFO - Started process (PID=4717) to work on /opt/airflow/dags/dagrundag1.py
[2024-02-27 09:19:34,469] {processor.py:613} INFO - Processing file /opt/airflow/dags/dagrundag1.py for tasks to queue
[2024-02-27 09:19:34,470] {logging_mixin.py:109} INFO - [2024-02-27 09:19:34,470] {dagbag.py:496} INFO - Filling up the DagBag from /opt/airflow/dags/dagrundag1.py
[2024-02-27 09:19:34,481] {processor.py:623} INFO - DAG(s) dict_keys(['dagrundag1']) retrieved from /opt/airflow/dags/dagrundag1.py
[2024-02-27 09:19:34,503] {logging_mixin.py:109} INFO - [2024-02-27 09:19:34,502] {dag.py:1834} INFO - Sync 1 DAGs
[2024-02-27 09:19:34,518] {logging_mixin.py:109} INFO - [2024-02-27 09:19:34,517] {dag.py:2287} INFO - Setting next_dagrun for dagrundag1 to None
[2024-02-27 09:19:34,532] {processor.py:170} INFO - Processing /opt/airflow/dags/dagrundag1.py took 0.070 seconds
[2024-02-27 09:19:39,024] {processor.py:162} INFO - Started process (PID=4728) to work on /opt/airflow/dags/dagrundag1.py
[2024-02-27 09:19:39,026] {processor.py:613} INFO - Processing file /opt/airflow/dags/dagrundag1.py for tasks to queue
[2024-02-27 09:19:39,028] {logging_mixin.py:109} INFO - [2024-02-27 09:19:39,028] {dagbag.py:496} INFO - Filling up the DagBag from /opt/airflow/dags/dagrundag1.py
[2024-02-27 09:19:39,034] {logging_mixin.py:109} INFO - [2024-02-27 09:19:39,031] {dagbag.py:329} ERROR - Failed to import: /opt/airflow/dags/dagrundag1.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/dagbag.py", line 326, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 674, in exec_module
  File "<frozen importlib._bootstrap_external>", line 781, in get_code
  File "<frozen importlib._bootstrap_external>", line 741, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dagrundag1.py", line 5
    from airflow
               ^
SyntaxError: invalid syntax
[2024-02-27 09:19:39,035] {processor.py:625} WARNING - No viable dags retrieved from /opt/airflow/dags/dagrundag1.py
[2024-02-27 09:19:39,064] {processor.py:170} INFO - Processing /opt/airflow/dags/dagrundag1.py took 0.045 seconds
[2024-02-27 09:19:42,546] {processor.py:162} INFO - Started process (PID=4739) to work on /opt/airflow/dags/dagrundag1.py
[2024-02-27 09:19:42,547] {processor.py:613} INFO - Processing file /opt/airflow/dags/dagrundag1.py for tasks to queue
[2024-02-27 09:19:42,548] {logging_mixin.py:109} INFO - [2024-02-27 09:19:42,548] {dagbag.py:496} INFO - Filling up the DagBag from /opt/airflow/dags/dagrundag1.py
[2024-02-27 09:19:42,551] {logging_mixin.py:109} INFO - [2024-02-27 09:19:42,550] {dagbag.py:329} ERROR - Failed to import: /opt/airflow/dags/dagrundag1.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/dagbag.py", line 326, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 674, in exec_module
  File "<frozen importlib._bootstrap_external>", line 781, in get_code
  File "<frozen importlib._bootstrap_external>", line 741, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dagrundag1.py", line 5
    from airflow.operators
                         ^
SyntaxError: invalid syntax
[2024-02-27 09:19:42,552] {processor.py:625} WARNING - No viable dags retrieved from /opt/airflow/dags/dagrundag1.py
[2024-02-27 09:19:42,580] {processor.py:170} INFO - Processing /opt/airflow/dags/dagrundag1.py took 0.041 seconds
[2024-02-27 09:19:45,555] {processor.py:162} INFO - Started process (PID=4740) to work on /opt/airflow/dags/dagrundag1.py
[2024-02-27 09:19:45,556] {processor.py:613} INFO - Processing file /opt/airflow/dags/dagrundag1.py for tasks to queue
[2024-02-27 09:19:45,558] {logging_mixin.py:109} INFO - [2024-02-27 09:19:45,558] {dagbag.py:496} INFO - Filling up the DagBag from /opt/airflow/dags/dagrundag1.py
[2024-02-27 09:19:45,561] {logging_mixin.py:109} INFO - [2024-02-27 09:19:45,560] {dagbag.py:329} ERROR - Failed to import: /opt/airflow/dags/dagrundag1.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/dagbag.py", line 326, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 674, in exec_module
  File "<frozen importlib._bootstrap_external>", line 781, in get_code
  File "<frozen importlib._bootstrap_external>", line 741, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dagrundag1.py", line 5
    from airflow.operators.dagrun_
                                 ^
SyntaxError: invalid syntax
[2024-02-27 09:19:45,561] {processor.py:625} WARNING - No viable dags retrieved from /opt/airflow/dags/dagrundag1.py
[2024-02-27 09:19:45,593] {processor.py:170} INFO - Processing /opt/airflow/dags/dagrundag1.py took 0.042 seconds
[2024-02-27 09:19:51,258] {processor.py:162} INFO - Started process (PID=4751) to work on /opt/airflow/dags/dagrundag1.py
[2024-02-27 09:19:51,263] {processor.py:613} INFO - Processing file /opt/airflow/dags/dagrundag1.py for tasks to queue
[2024-02-27 09:19:51,264] {logging_mixin.py:109} INFO - [2024-02-27 09:19:51,264] {dagbag.py:496} INFO - Filling up the DagBag from /opt/airflow/dags/dagrundag1.py
[2024-02-27 09:19:51,273] {logging_mixin.py:109} INFO - [2024-02-27 09:19:51,266] {dagbag.py:329} ERROR - Failed to import: /opt/airflow/dags/dagrundag1.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/dagbag.py", line 326, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 674, in exec_module
  File "<frozen importlib._bootstrap_external>", line 781, in get_code
  File "<frozen importlib._bootstrap_external>", line 741, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dagrundag1.py", line 5
    from airflow.operators.dagrun_
                                 ^
SyntaxError: invalid syntax
[2024-02-27 09:19:51,274] {processor.py:625} WARNING - No viable dags retrieved from /opt/airflow/dags/dagrundag1.py
[2024-02-27 09:19:51,301] {processor.py:170} INFO - Processing /opt/airflow/dags/dagrundag1.py took 0.046 seconds
[2024-02-27 09:19:54,790] {processor.py:162} INFO - Started process (PID=4762) to work on /opt/airflow/dags/dagrundag1.py
[2024-02-27 09:19:54,791] {processor.py:613} INFO - Processing file /opt/airflow/dags/dagrundag1.py for tasks to queue
[2024-02-27 09:19:54,792] {logging_mixin.py:109} INFO - [2024-02-27 09:19:54,792] {dagbag.py:496} INFO - Filling up the DagBag from /opt/airflow/dags/dagrundag1.py
[2024-02-27 09:19:54,832] {logging_mixin.py:109} INFO - [2024-02-27 09:19:54,831] {dagbag.py:329} ERROR - Failed to import: /opt/airflow/dags/dagrundag1.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/dagbag.py", line 326, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dagrundag1.py", line 5, in <module>
    from airflow.operators.dagrun_operator import Trigger
ImportError: cannot import name 'Trigger'
[2024-02-27 09:19:54,837] {processor.py:625} WARNING - No viable dags retrieved from /opt/airflow/dags/dagrundag1.py
[2024-02-27 09:19:54,876] {processor.py:170} INFO - Processing /opt/airflow/dags/dagrundag1.py took 0.091 seconds
[2024-02-27 09:19:57,398] {processor.py:162} INFO - Started process (PID=4772) to work on /opt/airflow/dags/dagrundag1.py
[2024-02-27 09:19:57,399] {processor.py:613} INFO - Processing file /opt/airflow/dags/dagrundag1.py for tasks to queue
[2024-02-27 09:19:57,400] {logging_mixin.py:109} INFO - [2024-02-27 09:19:57,400] {dagbag.py:496} INFO - Filling up the DagBag from /opt/airflow/dags/dagrundag1.py
[2024-02-27 09:19:57,416] {logging_mixin.py:109} INFO - [2024-02-27 09:19:57,415] {dagbag.py:329} ERROR - Failed to import: /opt/airflow/dags/dagrundag1.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/dagbag.py", line 326, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dagrundag1.py", line 5, in <module>
    from airflow.operators.dagrun_operator import TriggerDag
ImportError: cannot import name 'TriggerDag'
[2024-02-27 09:19:57,417] {processor.py:625} WARNING - No viable dags retrieved from /opt/airflow/dags/dagrundag1.py
[2024-02-27 09:19:57,439] {processor.py:170} INFO - Processing /opt/airflow/dags/dagrundag1.py took 0.045 seconds
[2024-02-27 09:20:00,491] {processor.py:162} INFO - Started process (PID=4774) to work on /opt/airflow/dags/dagrundag1.py
[2024-02-27 09:20:00,492] {processor.py:613} INFO - Processing file /opt/airflow/dags/dagrundag1.py for tasks to queue
[2024-02-27 09:20:00,493] {logging_mixin.py:109} INFO - [2024-02-27 09:20:00,493] {dagbag.py:496} INFO - Filling up the DagBag from /opt/airflow/dags/dagrundag1.py
[2024-02-27 09:20:00,506] {logging_mixin.py:109} INFO - [2024-02-27 09:20:00,505] {dagbag.py:329} ERROR - Failed to import: /opt/airflow/dags/dagrundag1.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/dagbag.py", line 326, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dagrundag1.py", line 5, in <module>
    from airflow.operators.dagrun_operator import TriggerDagRun
ImportError: cannot import name 'TriggerDagRun'
[2024-02-27 09:20:00,506] {processor.py:625} WARNING - No viable dags retrieved from /opt/airflow/dags/dagrundag1.py
[2024-02-27 09:20:00,531] {processor.py:170} INFO - Processing /opt/airflow/dags/dagrundag1.py took 0.043 seconds
[2024-02-27 09:20:02,517] {processor.py:162} INFO - Started process (PID=4775) to work on /opt/airflow/dags/dagrundag1.py
[2024-02-27 09:20:02,518] {processor.py:613} INFO - Processing file /opt/airflow/dags/dagrundag1.py for tasks to queue
[2024-02-27 09:20:02,519] {logging_mixin.py:109} INFO - [2024-02-27 09:20:02,519] {dagbag.py:496} INFO - Filling up the DagBag from /opt/airflow/dags/dagrundag1.py
[2024-02-27 09:20:02,540] {processor.py:623} INFO - DAG(s) dict_keys(['dagrundag1']) retrieved from /opt/airflow/dags/dagrundag1.py
[2024-02-27 09:20:02,578] {logging_mixin.py:109} INFO - [2024-02-27 09:20:02,578] {dag.py:1834} INFO - Sync 1 DAGs
[2024-02-27 09:20:02,595] {logging_mixin.py:109} INFO - [2024-02-27 09:20:02,595] {dag.py:2287} INFO - Setting next_dagrun for dagrundag1 to None
[2024-02-27 09:20:02,616] {processor.py:170} INFO - Processing /opt/airflow/dags/dagrundag1.py took 0.103 seconds
[2024-02-27 09:20:08,122] {processor.py:162} INFO - Started process (PID=4786) to work on /opt/airflow/dags/dagrundag1.py
[2024-02-27 09:20:08,124] {processor.py:613} INFO - Processing file /opt/airflow/dags/dagrundag1.py for tasks to queue
[2024-02-27 09:20:08,125] {logging_mixin.py:109} INFO - [2024-02-27 09:20:08,125] {dagbag.py:496} INFO - Filling up the DagBag from /opt/airflow/dags/dagrundag1.py
[2024-02-27 09:20:08,141] {processor.py:623} INFO - DAG(s) dict_keys(['dagrundag1']) retrieved from /opt/airflow/dags/dagrundag1.py
[2024-02-27 09:20:08,171] {logging_mixin.py:109} INFO - [2024-02-27 09:20:08,170] {dag.py:1834} INFO - Sync 1 DAGs
[2024-02-27 09:20:08,185] {logging_mixin.py:109} INFO - [2024-02-27 09:20:08,184] {dag.py:2287} INFO - Setting next_dagrun for dagrundag1 to None
[2024-02-27 09:20:08,199] {processor.py:170} INFO - Processing /opt/airflow/dags/dagrundag1.py took 0.083 seconds
[2024-02-27 09:20:13,751] {processor.py:162} INFO - Started process (PID=4797) to work on /opt/airflow/dags/dagrundag1.py
[2024-02-27 09:20:13,754] {processor.py:613} INFO - Processing file /opt/airflow/dags/dagrundag1.py for tasks to queue
[2024-02-27 09:20:13,756] {logging_mixin.py:109} INFO - [2024-02-27 09:20:13,755] {dagbag.py:496} INFO - Filling up the DagBag from /opt/airflow/dags/dagrundag1.py
[2024-02-27 09:20:13,775] {processor.py:623} INFO - DAG(s) dict_keys(['dagrundag1']) retrieved from /opt/airflow/dags/dagrundag1.py
[2024-02-27 09:20:13,812] {logging_mixin.py:109} INFO - [2024-02-27 09:20:13,812] {dag.py:1834} INFO - Sync 1 DAGs
[2024-02-27 09:20:13,840] {logging_mixin.py:109} INFO - [2024-02-27 09:20:13,840] {dag.py:2287} INFO - Setting next_dagrun for dagrundag1 to None
[2024-02-27 09:20:13,857] {processor.py:170} INFO - Processing /opt/airflow/dags/dagrundag1.py took 0.112 seconds
[2024-02-27 09:20:19,321] {processor.py:162} INFO - Started process (PID=4808) to work on /opt/airflow/dags/dagrundag1.py
[2024-02-27 09:20:19,322] {processor.py:613} INFO - Processing file /opt/airflow/dags/dagrundag1.py for tasks to queue
[2024-02-27 09:20:19,324] {logging_mixin.py:109} INFO - [2024-02-27 09:20:19,323] {dagbag.py:496} INFO - Filling up the DagBag from /opt/airflow/dags/dagrundag1.py
[2024-02-27 09:20:19,337] {processor.py:623} INFO - DAG(s) dict_keys(['dagrundag1']) retrieved from /opt/airflow/dags/dagrundag1.py
[2024-02-27 09:20:19,365] {logging_mixin.py:109} INFO - [2024-02-27 09:20:19,365] {dag.py:1834} INFO - Sync 1 DAGs
[2024-02-27 09:20:19,381] {logging_mixin.py:109} INFO - [2024-02-27 09:20:19,381] {dag.py:2287} INFO - Setting next_dagrun for dagrundag1 to None
[2024-02-27 09:20:19,395] {processor.py:170} INFO - Processing /opt/airflow/dags/dagrundag1.py took 0.078 seconds
[2024-02-27 09:20:24,872] {processor.py:162} INFO - Started process (PID=4819) to work on /opt/airflow/dags/dagrundag1.py
[2024-02-27 09:20:24,886] {processor.py:613} INFO - Processing file /opt/airflow/dags/dagrundag1.py for tasks to queue
[2024-02-27 09:20:24,888] {logging_mixin.py:109} INFO - [2024-02-27 09:20:24,887] {dagbag.py:496} INFO - Filling up the DagBag from /opt/airflow/dags/dagrundag1.py
[2024-02-27 09:20:24,916] {processor.py:623} INFO - DAG(s) dict_keys(['dagrundag1']) retrieved from /opt/airflow/dags/dagrundag1.py
[2024-02-27 09:20:24,973] {logging_mixin.py:109} INFO - [2024-02-27 09:20:24,972] {dag.py:1834} INFO - Sync 1 DAGs
[2024-02-27 09:20:24,997] {logging_mixin.py:109} INFO - [2024-02-27 09:20:24,997] {dag.py:2287} INFO - Setting next_dagrun for dagrundag1 to None
[2024-02-27 09:20:25,033] {processor.py:170} INFO - Processing /opt/airflow/dags/dagrundag1.py took 0.167 seconds
[2024-02-27 09:20:30,219] {processor.py:162} INFO - Started process (PID=4830) to work on /opt/airflow/dags/dagrundag1.py
[2024-02-27 09:20:30,220] {processor.py:613} INFO - Processing file /opt/airflow/dags/dagrundag1.py for tasks to queue
[2024-02-27 09:20:30,221] {logging_mixin.py:109} INFO - [2024-02-27 09:20:30,221] {dagbag.py:496} INFO - Filling up the DagBag from /opt/airflow/dags/dagrundag1.py
[2024-02-27 09:20:30,234] {processor.py:623} INFO - DAG(s) dict_keys(['dagrundag1']) retrieved from /opt/airflow/dags/dagrundag1.py
[2024-02-27 09:20:30,263] {logging_mixin.py:109} INFO - [2024-02-27 09:20:30,262] {dag.py:1834} INFO - Sync 1 DAGs
[2024-02-27 09:20:30,281] {logging_mixin.py:109} INFO - [2024-02-27 09:20:30,281] {dag.py:2287} INFO - Setting next_dagrun for dagrundag1 to None
[2024-02-27 09:20:30,295] {processor.py:170} INFO - Processing /opt/airflow/dags/dagrundag1.py took 0.081 seconds
[2024-02-27 09:20:35,766] {processor.py:162} INFO - Started process (PID=4841) to work on /opt/airflow/dags/dagrundag1.py
[2024-02-27 09:20:35,769] {processor.py:613} INFO - Processing file /opt/airflow/dags/dagrundag1.py for tasks to queue
[2024-02-27 09:20:35,770] {logging_mixin.py:109} INFO - [2024-02-27 09:20:35,770] {dagbag.py:496} INFO - Filling up the DagBag from /opt/airflow/dags/dagrundag1.py
[2024-02-27 09:20:35,784] {processor.py:623} INFO - DAG(s) dict_keys(['dagrundag1']) retrieved from /opt/airflow/dags/dagrundag1.py
[2024-02-27 09:20:35,817] {logging_mixin.py:109} INFO - [2024-02-27 09:20:35,817] {dag.py:1834} INFO - Sync 1 DAGs
[2024-02-27 09:20:35,834] {logging_mixin.py:109} INFO - [2024-02-27 09:20:35,834] {dag.py:2287} INFO - Setting next_dagrun for dagrundag1 to None
[2024-02-27 09:20:35,850] {processor.py:170} INFO - Processing /opt/airflow/dags/dagrundag1.py took 0.092 seconds
[2024-02-27 09:20:41,287] {processor.py:162} INFO - Started process (PID=4852) to work on /opt/airflow/dags/dagrundag1.py
[2024-02-27 09:20:41,292] {processor.py:613} INFO - Processing file /opt/airflow/dags/dagrundag1.py for tasks to queue
[2024-02-27 09:20:41,292] {logging_mixin.py:109} INFO - [2024-02-27 09:20:41,292] {dagbag.py:496} INFO - Filling up the DagBag from /opt/airflow/dags/dagrundag1.py
[2024-02-27 09:20:41,303] {processor.py:623} INFO - DAG(s) dict_keys(['dagrundag1']) retrieved from /opt/airflow/dags/dagrundag1.py
[2024-02-27 09:20:41,331] {logging_mixin.py:109} INFO - [2024-02-27 09:20:41,331] {dag.py:1834} INFO - Sync 1 DAGs
[2024-02-27 09:20:41,354] {logging_mixin.py:109} INFO - [2024-02-27 09:20:41,354] {dag.py:2287} INFO - Setting next_dagrun for dagrundag1 to None
[2024-02-27 09:20:41,367] {processor.py:170} INFO - Processing /opt/airflow/dags/dagrundag1.py took 0.085 seconds
[2024-02-27 09:20:46,757] {processor.py:162} INFO - Started process (PID=4863) to work on /opt/airflow/dags/dagrundag1.py
[2024-02-27 09:20:46,759] {processor.py:613} INFO - Processing file /opt/airflow/dags/dagrundag1.py for tasks to queue
[2024-02-27 09:20:46,760] {logging_mixin.py:109} INFO - [2024-02-27 09:20:46,760] {dagbag.py:496} INFO - Filling up the DagBag from /opt/airflow/dags/dagrundag1.py
[2024-02-27 09:20:46,778] {processor.py:623} INFO - DAG(s) dict_keys(['dagrundag1']) retrieved from /opt/airflow/dags/dagrundag1.py
[2024-02-27 09:20:46,815] {logging_mixin.py:109} INFO - [2024-02-27 09:20:46,814] {dag.py:1834} INFO - Sync 1 DAGs
[2024-02-27 09:20:46,834] {logging_mixin.py:109} INFO - [2024-02-27 09:20:46,834] {dag.py:2287} INFO - Setting next_dagrun for dagrundag1 to None
[2024-02-27 09:20:46,853] {processor.py:170} INFO - Processing /opt/airflow/dags/dagrundag1.py took 0.103 seconds
[2024-02-27 09:20:52,293] {processor.py:162} INFO - Started process (PID=4874) to work on /opt/airflow/dags/dagrundag1.py
[2024-02-27 09:20:52,296] {processor.py:613} INFO - Processing file /opt/airflow/dags/dagrundag1.py for tasks to queue
[2024-02-27 09:20:52,297] {logging_mixin.py:109} INFO - [2024-02-27 09:20:52,297] {dagbag.py:496} INFO - Filling up the DagBag from /opt/airflow/dags/dagrundag1.py
[2024-02-27 09:20:52,310] {processor.py:623} INFO - DAG(s) dict_keys(['dagrundag1']) retrieved from /opt/airflow/dags/dagrundag1.py
[2024-02-27 09:20:52,339] {logging_mixin.py:109} INFO - [2024-02-27 09:20:52,339] {dag.py:1834} INFO - Sync 1 DAGs
[2024-02-27 09:20:52,353] {logging_mixin.py:109} INFO - [2024-02-27 09:20:52,352] {dag.py:2287} INFO - Setting next_dagrun for dagrundag1 to None
[2024-02-27 09:20:52,365] {processor.py:170} INFO - Processing /opt/airflow/dags/dagrundag1.py took 0.077 seconds
[2024-02-27 09:20:57,802] {processor.py:162} INFO - Started process (PID=4885) to work on /opt/airflow/dags/dagrundag1.py
[2024-02-27 09:20:57,807] {processor.py:613} INFO - Processing file /opt/airflow/dags/dagrundag1.py for tasks to queue
[2024-02-27 09:20:57,808] {logging_mixin.py:109} INFO - [2024-02-27 09:20:57,808] {dagbag.py:496} INFO - Filling up the DagBag from /opt/airflow/dags/dagrundag1.py
[2024-02-27 09:20:57,818] {processor.py:623} INFO - DAG(s) dict_keys(['dagrundag1']) retrieved from /opt/airflow/dags/dagrundag1.py
[2024-02-27 09:20:57,847] {logging_mixin.py:109} INFO - [2024-02-27 09:20:57,847] {dag.py:1834} INFO - Sync 1 DAGs
[2024-02-27 09:20:57,863] {logging_mixin.py:109} INFO - [2024-02-27 09:20:57,862] {dag.py:2287} INFO - Setting next_dagrun for dagrundag1 to None
[2024-02-27 09:20:57,876] {processor.py:170} INFO - Processing /opt/airflow/dags/dagrundag1.py took 0.077 seconds
[2024-02-27 09:21:02,374] {processor.py:162} INFO - Started process (PID=4896) to work on /opt/airflow/dags/dagrundag1.py
[2024-02-27 09:21:02,375] {processor.py:613} INFO - Processing file /opt/airflow/dags/dagrundag1.py for tasks to queue
[2024-02-27 09:21:02,376] {logging_mixin.py:109} INFO - [2024-02-27 09:21:02,376] {dagbag.py:496} INFO - Filling up the DagBag from /opt/airflow/dags/dagrundag1.py
[2024-02-27 09:21:02,390] {logging_mixin.py:109} INFO - [2024-02-27 09:21:02,388] {dagbag.py:329} ERROR - Failed to import: /opt/airflow/dags/dagrundag1.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/dagbag.py", line 326, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dagrundag1.py", line 13, in <module>
    task2 = TriggerDagRunOperator(task_id="tsk2",bash_command="sleep 5",dag=dag )
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/baseoperator.py", line 167, in apply_defaults
    raise AirflowException(msg)
airflow.exceptions.AirflowException: Argument ['trigger_dag_id'] is required
[2024-02-27 09:21:02,391] {processor.py:625} WARNING - No viable dags retrieved from /opt/airflow/dags/dagrundag1.py
[2024-02-27 09:21:02,413] {processor.py:170} INFO - Processing /opt/airflow/dags/dagrundag1.py took 0.042 seconds
[2024-02-27 09:21:07,855] {processor.py:162} INFO - Started process (PID=4907) to work on /opt/airflow/dags/dagrundag1.py
[2024-02-27 09:21:07,856] {processor.py:613} INFO - Processing file /opt/airflow/dags/dagrundag1.py for tasks to queue
[2024-02-27 09:21:07,857] {logging_mixin.py:109} INFO - [2024-02-27 09:21:07,857] {dagbag.py:496} INFO - Filling up the DagBag from /opt/airflow/dags/dagrundag1.py
[2024-02-27 09:21:07,872] {logging_mixin.py:109} INFO - [2024-02-27 09:21:07,870] {dagbag.py:329} ERROR - Failed to import: /opt/airflow/dags/dagrundag1.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/dagbag.py", line 326, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dagrundag1.py", line 13, in <module>
    task2 = TriggerDagRunOperator(task_id="tsk2",bash_command="sleep 5",dag=dag )
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/baseoperator.py", line 167, in apply_defaults
    raise AirflowException(msg)
airflow.exceptions.AirflowException: Argument ['trigger_dag_id'] is required
[2024-02-27 09:21:07,872] {processor.py:625} WARNING - No viable dags retrieved from /opt/airflow/dags/dagrundag1.py
[2024-02-27 09:21:07,900] {processor.py:170} INFO - Processing /opt/airflow/dags/dagrundag1.py took 0.047 seconds
[2024-02-27 09:21:13,437] {processor.py:162} INFO - Started process (PID=4918) to work on /opt/airflow/dags/dagrundag1.py
[2024-02-27 09:21:13,444] {processor.py:613} INFO - Processing file /opt/airflow/dags/dagrundag1.py for tasks to queue
[2024-02-27 09:21:13,445] {logging_mixin.py:109} INFO - [2024-02-27 09:21:13,445] {dagbag.py:496} INFO - Filling up the DagBag from /opt/airflow/dags/dagrundag1.py
[2024-02-27 09:21:13,455] {logging_mixin.py:109} INFO - [2024-02-27 09:21:13,454] {dagbag.py:329} ERROR - Failed to import: /opt/airflow/dags/dagrundag1.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/dagbag.py", line 326, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dagrundag1.py", line 13, in <module>
    task2 = TriggerDagRunOperator(task_id="tsk2",bash_command="sleep 5",dag=dag )
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/baseoperator.py", line 167, in apply_defaults
    raise AirflowException(msg)
airflow.exceptions.AirflowException: Argument ['trigger_dag_id'] is required
[2024-02-27 09:21:13,455] {processor.py:625} WARNING - No viable dags retrieved from /opt/airflow/dags/dagrundag1.py
[2024-02-27 09:21:13,481] {processor.py:170} INFO - Processing /opt/airflow/dags/dagrundag1.py took 0.048 seconds
[2024-02-27 09:21:19,143] {processor.py:162} INFO - Started process (PID=4929) to work on /opt/airflow/dags/dagrundag1.py
[2024-02-27 09:21:19,146] {processor.py:613} INFO - Processing file /opt/airflow/dags/dagrundag1.py for tasks to queue
[2024-02-27 09:21:19,147] {logging_mixin.py:109} INFO - [2024-02-27 09:21:19,147] {dagbag.py:496} INFO - Filling up the DagBag from /opt/airflow/dags/dagrundag1.py
[2024-02-27 09:21:19,160] {logging_mixin.py:109} INFO - [2024-02-27 09:21:19,159] {dagbag.py:329} ERROR - Failed to import: /opt/airflow/dags/dagrundag1.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/dagbag.py", line 326, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dagrundag1.py", line 13, in <module>
    task2 = TriggerDagRunOperator(task_id="tsk2",bash_command="sleep 5",dag=dag )
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/baseoperator.py", line 167, in apply_defaults
    raise AirflowException(msg)
airflow.exceptions.AirflowException: Argument ['trigger_dag_id'] is required
[2024-02-27 09:21:19,161] {processor.py:625} WARNING - No viable dags retrieved from /opt/airflow/dags/dagrundag1.py
[2024-02-27 09:21:19,191] {processor.py:170} INFO - Processing /opt/airflow/dags/dagrundag1.py took 0.052 seconds
[2024-02-27 09:21:24,988] {processor.py:162} INFO - Started process (PID=4940) to work on /opt/airflow/dags/dagrundag1.py
[2024-02-27 09:21:24,992] {processor.py:613} INFO - Processing file /opt/airflow/dags/dagrundag1.py for tasks to queue
[2024-02-27 09:21:24,993] {logging_mixin.py:109} INFO - [2024-02-27 09:21:24,993] {dagbag.py:496} INFO - Filling up the DagBag from /opt/airflow/dags/dagrundag1.py
[2024-02-27 09:21:25,008] {logging_mixin.py:109} INFO - [2024-02-27 09:21:25,007] {dagbag.py:329} ERROR - Failed to import: /opt/airflow/dags/dagrundag1.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/dagbag.py", line 326, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dagrundag1.py", line 13, in <module>
    task2 = TriggerDagRunOperator(task_id="tsk2",trigger_dag="sleep 5",dag=dag )
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/baseoperator.py", line 167, in apply_defaults
    raise AirflowException(msg)
airflow.exceptions.AirflowException: Argument ['trigger_dag_id'] is required
[2024-02-27 09:21:25,009] {processor.py:625} WARNING - No viable dags retrieved from /opt/airflow/dags/dagrundag1.py
[2024-02-27 09:21:25,032] {processor.py:170} INFO - Processing /opt/airflow/dags/dagrundag1.py took 0.049 seconds
[2024-02-27 09:21:28,567] {processor.py:162} INFO - Started process (PID=4951) to work on /opt/airflow/dags/dagrundag1.py
[2024-02-27 09:21:28,569] {processor.py:613} INFO - Processing file /opt/airflow/dags/dagrundag1.py for tasks to queue
[2024-02-27 09:21:28,569] {logging_mixin.py:109} INFO - [2024-02-27 09:21:28,569] {dagbag.py:496} INFO - Filling up the DagBag from /opt/airflow/dags/dagrundag1.py
[2024-02-27 09:21:28,587] {logging_mixin.py:109} INFO - [2024-02-27 09:21:28,585] {dagbag.py:329} ERROR - Failed to import: /opt/airflow/dags/dagrundag1.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/dagbag.py", line 326, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dagrundag1.py", line 13, in <module>
    task2 = TriggerDagRunOperator(task_id="tsk2",trigger_dagid="sleep 5",dag=dag )
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/baseoperator.py", line 167, in apply_defaults
    raise AirflowException(msg)
airflow.exceptions.AirflowException: Argument ['trigger_dag_id'] is required
[2024-02-27 09:21:28,587] {processor.py:625} WARNING - No viable dags retrieved from /opt/airflow/dags/dagrundag1.py
[2024-02-27 09:21:28,615] {processor.py:170} INFO - Processing /opt/airflow/dags/dagrundag1.py took 0.051 seconds
[2024-02-27 09:21:31,620] {processor.py:162} INFO - Started process (PID=4952) to work on /opt/airflow/dags/dagrundag1.py
[2024-02-27 09:21:31,622] {processor.py:613} INFO - Processing file /opt/airflow/dags/dagrundag1.py for tasks to queue
[2024-02-27 09:21:31,623] {logging_mixin.py:109} INFO - [2024-02-27 09:21:31,623] {dagbag.py:496} INFO - Filling up the DagBag from /opt/airflow/dags/dagrundag1.py
[2024-02-27 09:21:31,634] {logging_mixin.py:109} INFO - [2024-02-27 09:21:31,633] {dagbag.py:329} ERROR - Failed to import: /opt/airflow/dags/dagrundag1.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/dagbag.py", line 326, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dagrundag1.py", line 13, in <module>
    task2 = TriggerDagRunOperator(task_id="tsk2",trigger_dagid="",dag=dag )
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/baseoperator.py", line 167, in apply_defaults
    raise AirflowException(msg)
airflow.exceptions.AirflowException: Argument ['trigger_dag_id'] is required
[2024-02-27 09:21:31,635] {processor.py:625} WARNING - No viable dags retrieved from /opt/airflow/dags/dagrundag1.py
[2024-02-27 09:21:31,662] {processor.py:170} INFO - Processing /opt/airflow/dags/dagrundag1.py took 0.046 seconds
[2024-02-27 09:21:36,167] {processor.py:162} INFO - Started process (PID=4963) to work on /opt/airflow/dags/dagrundag1.py
[2024-02-27 09:21:36,169] {processor.py:613} INFO - Processing file /opt/airflow/dags/dagrundag1.py for tasks to queue
[2024-02-27 09:21:36,170] {logging_mixin.py:109} INFO - [2024-02-27 09:21:36,170] {dagbag.py:496} INFO - Filling up the DagBag from /opt/airflow/dags/dagrundag1.py
[2024-02-27 09:21:36,183] {logging_mixin.py:109} INFO - [2024-02-27 09:21:36,181] {dagbag.py:329} ERROR - Failed to import: /opt/airflow/dags/dagrundag1.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/dagbag.py", line 326, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dagrundag1.py", line 13, in <module>
    task2 = TriggerDagRunOperator(task_id="tsk2",trigger_dagid="da",dag=dag )
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/baseoperator.py", line 167, in apply_defaults
    raise AirflowException(msg)
airflow.exceptions.AirflowException: Argument ['trigger_dag_id'] is required
[2024-02-27 09:21:36,184] {processor.py:625} WARNING - No viable dags retrieved from /opt/airflow/dags/dagrundag1.py
[2024-02-27 09:21:36,209] {processor.py:170} INFO - Processing /opt/airflow/dags/dagrundag1.py took 0.045 seconds
[2024-02-27 09:21:38,805] {processor.py:162} INFO - Started process (PID=4974) to work on /opt/airflow/dags/dagrundag1.py
[2024-02-27 09:21:38,806] {processor.py:613} INFO - Processing file /opt/airflow/dags/dagrundag1.py for tasks to queue
[2024-02-27 09:21:38,806] {logging_mixin.py:109} INFO - [2024-02-27 09:21:38,806] {dagbag.py:496} INFO - Filling up the DagBag from /opt/airflow/dags/dagrundag1.py
[2024-02-27 09:21:38,821] {logging_mixin.py:109} INFO - [2024-02-27 09:21:38,819] {dagbag.py:329} ERROR - Failed to import: /opt/airflow/dags/dagrundag1.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/dagbag.py", line 326, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dagrundag1.py", line 13, in <module>
    task2 = TriggerDagRunOperator(task_id="tsk2",trigger_dagid="dagrunda",dag=dag )
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/baseoperator.py", line 167, in apply_defaults
    raise AirflowException(msg)
airflow.exceptions.AirflowException: Argument ['trigger_dag_id'] is required
[2024-02-27 09:21:38,821] {processor.py:625} WARNING - No viable dags retrieved from /opt/airflow/dags/dagrundag1.py
[2024-02-27 09:21:38,846] {processor.py:170} INFO - Processing /opt/airflow/dags/dagrundag1.py took 0.045 seconds
[2024-02-27 09:21:41,819] {processor.py:162} INFO - Started process (PID=4975) to work on /opt/airflow/dags/dagrundag1.py
[2024-02-27 09:21:41,824] {processor.py:613} INFO - Processing file /opt/airflow/dags/dagrundag1.py for tasks to queue
[2024-02-27 09:21:41,825] {logging_mixin.py:109} INFO - [2024-02-27 09:21:41,825] {dagbag.py:496} INFO - Filling up the DagBag from /opt/airflow/dags/dagrundag1.py
[2024-02-27 09:21:41,837] {logging_mixin.py:109} INFO - [2024-02-27 09:21:41,836] {dagbag.py:329} ERROR - Failed to import: /opt/airflow/dags/dagrundag1.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/dagbag.py", line 326, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dagrundag1.py", line 13, in <module>
    task2 = TriggerDagRunOperator(task_id="tsk2",trigger_dagid="dagrundag2",dag=dag )
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/baseoperator.py", line 167, in apply_defaults
    raise AirflowException(msg)
airflow.exceptions.AirflowException: Argument ['trigger_dag_id'] is required
[2024-02-27 09:21:41,838] {processor.py:625} WARNING - No viable dags retrieved from /opt/airflow/dags/dagrundag1.py
[2024-02-27 09:21:41,865] {processor.py:170} INFO - Processing /opt/airflow/dags/dagrundag1.py took 0.049 seconds
[2024-02-27 09:21:45,383] {processor.py:162} INFO - Started process (PID=4986) to work on /opt/airflow/dags/dagrundag1.py
[2024-02-27 09:21:45,384] {processor.py:613} INFO - Processing file /opt/airflow/dags/dagrundag1.py for tasks to queue
[2024-02-27 09:21:45,385] {logging_mixin.py:109} INFO - [2024-02-27 09:21:45,385] {dagbag.py:496} INFO - Filling up the DagBag from /opt/airflow/dags/dagrundag1.py
[2024-02-27 09:21:45,395] {logging_mixin.py:109} INFO - [2024-02-27 09:21:45,394] {dagbag.py:329} ERROR - Failed to import: /opt/airflow/dags/dagrundag1.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/dagbag.py", line 326, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dagrundag1.py", line 13, in <module>
    task2 = TriggerDagRunOperator(task_id="tsk2",trigger_dagid="dagrundag2",dag=dag )
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/baseoperator.py", line 167, in apply_defaults
    raise AirflowException(msg)
airflow.exceptions.AirflowException: Argument ['trigger_dag_id'] is required
[2024-02-27 09:21:45,396] {processor.py:625} WARNING - No viable dags retrieved from /opt/airflow/dags/dagrundag1.py
[2024-02-27 09:21:45,421] {processor.py:170} INFO - Processing /opt/airflow/dags/dagrundag1.py took 0.045 seconds
[2024-02-27 09:21:51,116] {processor.py:162} INFO - Started process (PID=4997) to work on /opt/airflow/dags/dagrundag1.py
[2024-02-27 09:21:51,119] {processor.py:613} INFO - Processing file /opt/airflow/dags/dagrundag1.py for tasks to queue
[2024-02-27 09:21:51,121] {logging_mixin.py:109} INFO - [2024-02-27 09:21:51,120] {dagbag.py:496} INFO - Filling up the DagBag from /opt/airflow/dags/dagrundag1.py
[2024-02-27 09:21:51,133] {logging_mixin.py:109} INFO - [2024-02-27 09:21:51,131] {dagbag.py:329} ERROR - Failed to import: /opt/airflow/dags/dagrundag1.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/dagbag.py", line 326, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dagrundag1.py", line 13, in <module>
    task2 = TriggerDagRunOperator(task_id="tsk2",trigger_dagid="dagrundag2",dag=dag )
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/baseoperator.py", line 167, in apply_defaults
    raise AirflowException(msg)
airflow.exceptions.AirflowException: Argument ['trigger_dag_id'] is required
[2024-02-27 09:21:51,134] {processor.py:625} WARNING - No viable dags retrieved from /opt/airflow/dags/dagrundag1.py
[2024-02-27 09:21:51,168] {processor.py:170} INFO - Processing /opt/airflow/dags/dagrundag1.py took 0.056 seconds
[2024-02-27 09:21:56,611] {processor.py:162} INFO - Started process (PID=5008) to work on /opt/airflow/dags/dagrundag1.py
[2024-02-27 09:21:56,613] {processor.py:613} INFO - Processing file /opt/airflow/dags/dagrundag1.py for tasks to queue
[2024-02-27 09:21:56,615] {logging_mixin.py:109} INFO - [2024-02-27 09:21:56,615] {dagbag.py:496} INFO - Filling up the DagBag from /opt/airflow/dags/dagrundag1.py
[2024-02-27 09:21:56,626] {logging_mixin.py:109} INFO - [2024-02-27 09:21:56,625] {dagbag.py:329} ERROR - Failed to import: /opt/airflow/dags/dagrundag1.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/dagbag.py", line 326, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dagrundag1.py", line 13, in <module>
    task2 = TriggerDagRunOperator(task_id="tsk2",trigger_dagid="dagrundag2",dag=dag )
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/baseoperator.py", line 167, in apply_defaults
    raise AirflowException(msg)
airflow.exceptions.AirflowException: Argument ['trigger_dag_id'] is required
[2024-02-27 09:21:56,627] {processor.py:625} WARNING - No viable dags retrieved from /opt/airflow/dags/dagrundag1.py
[2024-02-27 09:21:56,654] {processor.py:170} INFO - Processing /opt/airflow/dags/dagrundag1.py took 0.047 seconds
[2024-02-27 09:22:02,127] {processor.py:162} INFO - Started process (PID=5019) to work on /opt/airflow/dags/dagrundag1.py
[2024-02-27 09:22:02,129] {processor.py:613} INFO - Processing file /opt/airflow/dags/dagrundag1.py for tasks to queue
[2024-02-27 09:22:02,131] {logging_mixin.py:109} INFO - [2024-02-27 09:22:02,131] {dagbag.py:496} INFO - Filling up the DagBag from /opt/airflow/dags/dagrundag1.py
[2024-02-27 09:22:02,146] {logging_mixin.py:109} INFO - [2024-02-27 09:22:02,142] {dagbag.py:329} ERROR - Failed to import: /opt/airflow/dags/dagrundag1.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/dagbag.py", line 326, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dagrundag1.py", line 13, in <module>
    task2 = TriggerDagRunOperator(task_id="tsk2",trigger_dagid="dagrundag2",dag=dag )
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/baseoperator.py", line 167, in apply_defaults
    raise AirflowException(msg)
airflow.exceptions.AirflowException: Argument ['trigger_dag_id'] is required
[2024-02-27 09:22:02,147] {processor.py:625} WARNING - No viable dags retrieved from /opt/airflow/dags/dagrundag1.py
[2024-02-27 09:22:02,174] {processor.py:170} INFO - Processing /opt/airflow/dags/dagrundag1.py took 0.050 seconds
[2024-02-27 09:22:07,585] {processor.py:162} INFO - Started process (PID=5030) to work on /opt/airflow/dags/dagrundag1.py
[2024-02-27 09:22:07,586] {processor.py:613} INFO - Processing file /opt/airflow/dags/dagrundag1.py for tasks to queue
[2024-02-27 09:22:07,588] {logging_mixin.py:109} INFO - [2024-02-27 09:22:07,588] {dagbag.py:496} INFO - Filling up the DagBag from /opt/airflow/dags/dagrundag1.py
[2024-02-27 09:22:07,600] {logging_mixin.py:109} INFO - [2024-02-27 09:22:07,598] {dagbag.py:329} ERROR - Failed to import: /opt/airflow/dags/dagrundag1.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/dagbag.py", line 326, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dagrundag1.py", line 13, in <module>
    task2 = TriggerDagRunOperator(task_id="tsk2",trigger_dagid="dagrundag2",dag=dag )
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/baseoperator.py", line 167, in apply_defaults
    raise AirflowException(msg)
airflow.exceptions.AirflowException: Argument ['trigger_dag_id'] is required
[2024-02-27 09:22:07,601] {processor.py:625} WARNING - No viable dags retrieved from /opt/airflow/dags/dagrundag1.py
[2024-02-27 09:22:07,627] {processor.py:170} INFO - Processing /opt/airflow/dags/dagrundag1.py took 0.046 seconds
[2024-02-27 09:22:13,186] {processor.py:162} INFO - Started process (PID=5041) to work on /opt/airflow/dags/dagrundag1.py
[2024-02-27 09:22:13,191] {processor.py:613} INFO - Processing file /opt/airflow/dags/dagrundag1.py for tasks to queue
[2024-02-27 09:22:13,192] {logging_mixin.py:109} INFO - [2024-02-27 09:22:13,192] {dagbag.py:496} INFO - Filling up the DagBag from /opt/airflow/dags/dagrundag1.py
[2024-02-27 09:22:13,207] {logging_mixin.py:109} INFO - [2024-02-27 09:22:13,205] {dagbag.py:329} ERROR - Failed to import: /opt/airflow/dags/dagrundag1.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/dagbag.py", line 326, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dagrundag1.py", line 13, in <module>
    task2 = TriggerDagRunOperator(task_id="tsk2",trigger_dagid="dagrundag2",dag=dag )
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/baseoperator.py", line 167, in apply_defaults
    raise AirflowException(msg)
airflow.exceptions.AirflowException: Argument ['trigger_dag_id'] is required
[2024-02-27 09:22:13,208] {processor.py:625} WARNING - No viable dags retrieved from /opt/airflow/dags/dagrundag1.py
[2024-02-27 09:22:13,233] {processor.py:170} INFO - Processing /opt/airflow/dags/dagrundag1.py took 0.053 seconds
[2024-02-27 09:22:18,734] {processor.py:162} INFO - Started process (PID=5052) to work on /opt/airflow/dags/dagrundag1.py
[2024-02-27 09:22:18,735] {processor.py:613} INFO - Processing file /opt/airflow/dags/dagrundag1.py for tasks to queue
[2024-02-27 09:22:18,736] {logging_mixin.py:109} INFO - [2024-02-27 09:22:18,736] {dagbag.py:496} INFO - Filling up the DagBag from /opt/airflow/dags/dagrundag1.py
[2024-02-27 09:22:18,748] {logging_mixin.py:109} INFO - [2024-02-27 09:22:18,746] {dagbag.py:329} ERROR - Failed to import: /opt/airflow/dags/dagrundag1.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/dagbag.py", line 326, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dagrundag1.py", line 13, in <module>
    task2 = TriggerDagRunOperator(task_id="tsk2",trigger_dagid="dagrundag2",dag=dag )
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/baseoperator.py", line 167, in apply_defaults
    raise AirflowException(msg)
airflow.exceptions.AirflowException: Argument ['trigger_dag_id'] is required
[2024-02-27 09:22:18,748] {processor.py:625} WARNING - No viable dags retrieved from /opt/airflow/dags/dagrundag1.py
[2024-02-27 09:22:18,774] {processor.py:170} INFO - Processing /opt/airflow/dags/dagrundag1.py took 0.043 seconds
[2024-02-27 09:22:22,909] {processor.py:162} INFO - Started process (PID=5056) to work on /opt/airflow/dags/dagrundag1.py
[2024-02-27 09:22:22,911] {processor.py:613} INFO - Processing file /opt/airflow/dags/dagrundag1.py for tasks to queue
[2024-02-27 09:22:22,911] {logging_mixin.py:109} INFO - [2024-02-27 09:22:22,911] {dagbag.py:496} INFO - Filling up the DagBag from /opt/airflow/dags/dagrundag1.py
[2024-02-27 09:22:22,924] {logging_mixin.py:109} INFO - [2024-02-27 09:22:22,921] {dagbag.py:329} ERROR - Failed to import: /opt/airflow/dags/dagrundag1.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/dagbag.py", line 326, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dagrundag1.py", line 13, in <module>
    task2 = TriggerDagRunOperator(task_id="tsk2",trigger_dagid="dagrundag2",dag=dag )
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/baseoperator.py", line 167, in apply_defaults
    raise AirflowException(msg)
airflow.exceptions.AirflowException: Argument ['trigger_dag_id'] is required
[2024-02-27 09:22:22,925] {processor.py:625} WARNING - No viable dags retrieved from /opt/airflow/dags/dagrundag1.py
[2024-02-27 09:22:22,950] {processor.py:170} INFO - Processing /opt/airflow/dags/dagrundag1.py took 0.045 seconds
[2024-02-27 09:22:28,326] {processor.py:162} INFO - Started process (PID=5065) to work on /opt/airflow/dags/dagrundag1.py
[2024-02-27 09:22:28,328] {processor.py:613} INFO - Processing file /opt/airflow/dags/dagrundag1.py for tasks to queue
[2024-02-27 09:22:28,330] {logging_mixin.py:109} INFO - [2024-02-27 09:22:28,330] {dagbag.py:496} INFO - Filling up the DagBag from /opt/airflow/dags/dagrundag1.py
[2024-02-27 09:22:28,358] {logging_mixin.py:109} INFO - [2024-02-27 09:22:28,347] {dagbag.py:329} ERROR - Failed to import: /opt/airflow/dags/dagrundag1.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/dagbag.py", line 326, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dagrundag1.py", line 13, in <module>
    task2 = TriggerDagRunOperator(task_id="tsk2",trigger_dagid="dagrundag2",dag=dag )
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/baseoperator.py", line 167, in apply_defaults
    raise AirflowException(msg)
airflow.exceptions.AirflowException: Argument ['trigger_dag_id'] is required
[2024-02-27 09:22:28,360] {processor.py:625} WARNING - No viable dags retrieved from /opt/airflow/dags/dagrundag1.py
[2024-02-27 09:22:28,432] {processor.py:170} INFO - Processing /opt/airflow/dags/dagrundag1.py took 0.111 seconds
[2024-02-27 09:22:33,535] {processor.py:162} INFO - Started process (PID=5076) to work on /opt/airflow/dags/dagrundag1.py
[2024-02-27 09:22:33,538] {processor.py:613} INFO - Processing file /opt/airflow/dags/dagrundag1.py for tasks to queue
[2024-02-27 09:22:33,540] {logging_mixin.py:109} INFO - [2024-02-27 09:22:33,540] {dagbag.py:496} INFO - Filling up the DagBag from /opt/airflow/dags/dagrundag1.py
[2024-02-27 09:22:33,572] {logging_mixin.py:109} INFO - [2024-02-27 09:22:33,566] {dagbag.py:329} ERROR - Failed to import: /opt/airflow/dags/dagrundag1.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/dagbag.py", line 326, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dagrundag1.py", line 13, in <module>
    task2 = TriggerDagRunOperator(task_id="tsk2",trigger_dagid="dagrundag2",dag=dag )
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/baseoperator.py", line 167, in apply_defaults
    raise AirflowException(msg)
airflow.exceptions.AirflowException: Argument ['trigger_dag_id'] is required
[2024-02-27 09:22:33,573] {processor.py:625} WARNING - No viable dags retrieved from /opt/airflow/dags/dagrundag1.py
[2024-02-27 09:22:33,610] {processor.py:170} INFO - Processing /opt/airflow/dags/dagrundag1.py took 0.082 seconds
[2024-02-27 09:22:34,175] {processor.py:162} INFO - Started process (PID=5082) to work on /opt/airflow/dags/dagrundag1.py
[2024-02-27 09:22:34,177] {processor.py:613} INFO - Processing file /opt/airflow/dags/dagrundag1.py for tasks to queue
[2024-02-27 09:22:34,182] {logging_mixin.py:109} INFO - [2024-02-27 09:22:34,182] {dagbag.py:496} INFO - Filling up the DagBag from /opt/airflow/dags/dagrundag1.py
[2024-02-27 09:22:34,193] {logging_mixin.py:109} INFO - [2024-02-27 09:22:34,192] {dagbag.py:329} ERROR - Failed to import: /opt/airflow/dags/dagrundag1.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/dagbag.py", line 326, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dagrundag1.py", line 13, in <module>
    task2 = TriggerDagRunOperator(task_id="tsk2",trigger_dagid="dagrundag2",dag=dag )
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/baseoperator.py", line 167, in apply_defaults
    raise AirflowException(msg)
airflow.exceptions.AirflowException: Argument ['trigger_dag_id'] is required
[2024-02-27 09:22:34,194] {processor.py:625} WARNING - No viable dags retrieved from /opt/airflow/dags/dagrundag1.py
[2024-02-27 09:22:34,223] {processor.py:170} INFO - Processing /opt/airflow/dags/dagrundag1.py took 0.051 seconds
[2024-02-27 09:22:39,565] {processor.py:162} INFO - Started process (PID=5088) to work on /opt/airflow/dags/dagrundag1.py
[2024-02-27 09:22:39,568] {processor.py:613} INFO - Processing file /opt/airflow/dags/dagrundag1.py for tasks to queue
[2024-02-27 09:22:39,569] {logging_mixin.py:109} INFO - [2024-02-27 09:22:39,569] {dagbag.py:496} INFO - Filling up the DagBag from /opt/airflow/dags/dagrundag1.py
[2024-02-27 09:22:39,582] {logging_mixin.py:109} INFO - [2024-02-27 09:22:39,580] {dagbag.py:329} ERROR - Failed to import: /opt/airflow/dags/dagrundag1.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/dagbag.py", line 326, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dagrundag1.py", line 13, in <module>
    task2 = TriggerDagRunOperator(task_id="tsk2",trigger_dagid="dagrundag2",dag=dag )
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/baseoperator.py", line 167, in apply_defaults
    raise AirflowException(msg)
airflow.exceptions.AirflowException: Argument ['trigger_dag_id'] is required
[2024-02-27 09:22:39,582] {processor.py:625} WARNING - No viable dags retrieved from /opt/airflow/dags/dagrundag1.py
[2024-02-27 09:22:39,607] {processor.py:170} INFO - Processing /opt/airflow/dags/dagrundag1.py took 0.046 seconds
[2024-02-27 09:22:45,151] {processor.py:162} INFO - Started process (PID=5099) to work on /opt/airflow/dags/dagrundag1.py
[2024-02-27 09:22:45,158] {processor.py:613} INFO - Processing file /opt/airflow/dags/dagrundag1.py for tasks to queue
[2024-02-27 09:22:45,160] {logging_mixin.py:109} INFO - [2024-02-27 09:22:45,160] {dagbag.py:496} INFO - Filling up the DagBag from /opt/airflow/dags/dagrundag1.py
[2024-02-27 09:22:45,174] {logging_mixin.py:109} INFO - [2024-02-27 09:22:45,172] {dagbag.py:329} ERROR - Failed to import: /opt/airflow/dags/dagrundag1.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/dagbag.py", line 326, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dagrundag1.py", line 13, in <module>
    task2 = TriggerDagRunOperator(task_id="tsk2",trigger_dagid="dagrundag2",dag=dag )
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/baseoperator.py", line 167, in apply_defaults
    raise AirflowException(msg)
airflow.exceptions.AirflowException: Argument ['trigger_dag_id'] is required
[2024-02-27 09:22:45,175] {processor.py:625} WARNING - No viable dags retrieved from /opt/airflow/dags/dagrundag1.py
[2024-02-27 09:22:45,203] {processor.py:170} INFO - Processing /opt/airflow/dags/dagrundag1.py took 0.057 seconds
[2024-02-27 09:22:50,685] {processor.py:162} INFO - Started process (PID=5110) to work on /opt/airflow/dags/dagrundag1.py
[2024-02-27 09:22:50,686] {processor.py:613} INFO - Processing file /opt/airflow/dags/dagrundag1.py for tasks to queue
[2024-02-27 09:22:50,688] {logging_mixin.py:109} INFO - [2024-02-27 09:22:50,687] {dagbag.py:496} INFO - Filling up the DagBag from /opt/airflow/dags/dagrundag1.py
[2024-02-27 09:22:50,698] {logging_mixin.py:109} INFO - [2024-02-27 09:22:50,697] {dagbag.py:329} ERROR - Failed to import: /opt/airflow/dags/dagrundag1.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/dagbag.py", line 326, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dagrundag1.py", line 13, in <module>
    task2 = TriggerDagRunOperator(task_id="tsk2",trigger_dagid="dagrundag2",dag=dag )
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/baseoperator.py", line 167, in apply_defaults
    raise AirflowException(msg)
airflow.exceptions.AirflowException: Argument ['trigger_dag_id'] is required
[2024-02-27 09:22:50,699] {processor.py:625} WARNING - No viable dags retrieved from /opt/airflow/dags/dagrundag1.py
[2024-02-27 09:22:50,725] {processor.py:170} INFO - Processing /opt/airflow/dags/dagrundag1.py took 0.044 seconds
[2024-02-27 09:22:54,288] {processor.py:162} INFO - Started process (PID=5120) to work on /opt/airflow/dags/dagrundag1.py
[2024-02-27 09:22:54,289] {processor.py:613} INFO - Processing file /opt/airflow/dags/dagrundag1.py for tasks to queue
[2024-02-27 09:22:54,290] {logging_mixin.py:109} INFO - [2024-02-27 09:22:54,290] {dagbag.py:496} INFO - Filling up the DagBag from /opt/airflow/dags/dagrundag1.py
[2024-02-27 09:22:54,300] {processor.py:623} INFO - DAG(s) dict_keys(['dagrundag1']) retrieved from /opt/airflow/dags/dagrundag1.py
[2024-02-27 09:22:54,547] {logging_mixin.py:109} INFO - [2024-02-27 09:22:54,547] {dag.py:1834} INFO - Sync 1 DAGs
[2024-02-27 09:22:54,560] {logging_mixin.py:109} INFO - [2024-02-27 09:22:54,560] {dag.py:2287} INFO - Setting next_dagrun for dagrundag1 to None
[2024-02-27 09:22:54,579] {processor.py:170} INFO - Processing /opt/airflow/dags/dagrundag1.py took 0.295 seconds
[2024-02-27 09:22:59,706] {processor.py:162} INFO - Started process (PID=5131) to work on /opt/airflow/dags/dagrundag1.py
[2024-02-27 09:22:59,712] {processor.py:613} INFO - Processing file /opt/airflow/dags/dagrundag1.py for tasks to queue
[2024-02-27 09:22:59,713] {logging_mixin.py:109} INFO - [2024-02-27 09:22:59,712] {dagbag.py:496} INFO - Filling up the DagBag from /opt/airflow/dags/dagrundag1.py
[2024-02-27 09:22:59,723] {processor.py:623} INFO - DAG(s) dict_keys(['dagrundag1']) retrieved from /opt/airflow/dags/dagrundag1.py
[2024-02-27 09:22:59,742] {logging_mixin.py:109} INFO - [2024-02-27 09:22:59,742] {dag.py:1834} INFO - Sync 1 DAGs
[2024-02-27 09:22:59,757] {logging_mixin.py:109} INFO - [2024-02-27 09:22:59,756] {dag.py:2287} INFO - Setting next_dagrun for dagrundag1 to None
[2024-02-27 09:22:59,767] {processor.py:170} INFO - Processing /opt/airflow/dags/dagrundag1.py took 0.067 seconds
[2024-02-27 09:23:05,335] {processor.py:162} INFO - Started process (PID=5142) to work on /opt/airflow/dags/dagrundag1.py
[2024-02-27 09:23:05,336] {processor.py:613} INFO - Processing file /opt/airflow/dags/dagrundag1.py for tasks to queue
[2024-02-27 09:23:05,337] {logging_mixin.py:109} INFO - [2024-02-27 09:23:05,337] {dagbag.py:496} INFO - Filling up the DagBag from /opt/airflow/dags/dagrundag1.py
[2024-02-27 09:23:05,351] {processor.py:623} INFO - DAG(s) dict_keys(['dagrundag1']) retrieved from /opt/airflow/dags/dagrundag1.py
[2024-02-27 09:23:05,374] {logging_mixin.py:109} INFO - [2024-02-27 09:23:05,374] {dag.py:1834} INFO - Sync 1 DAGs
[2024-02-27 09:23:05,392] {logging_mixin.py:109} INFO - [2024-02-27 09:23:05,392] {dag.py:2287} INFO - Setting next_dagrun for dagrundag1 to None
[2024-02-27 09:23:05,404] {processor.py:170} INFO - Processing /opt/airflow/dags/dagrundag1.py took 0.075 seconds
[2024-02-27 09:23:10,947] {processor.py:162} INFO - Started process (PID=5153) to work on /opt/airflow/dags/dagrundag1.py
[2024-02-27 09:23:10,949] {processor.py:613} INFO - Processing file /opt/airflow/dags/dagrundag1.py for tasks to queue
[2024-02-27 09:23:10,950] {logging_mixin.py:109} INFO - [2024-02-27 09:23:10,950] {dagbag.py:496} INFO - Filling up the DagBag from /opt/airflow/dags/dagrundag1.py
[2024-02-27 09:23:10,960] {processor.py:623} INFO - DAG(s) dict_keys(['dagrundag1']) retrieved from /opt/airflow/dags/dagrundag1.py
[2024-02-27 09:23:10,983] {logging_mixin.py:109} INFO - [2024-02-27 09:23:10,983] {dag.py:1834} INFO - Sync 1 DAGs
[2024-02-27 09:23:10,996] {logging_mixin.py:109} INFO - [2024-02-27 09:23:10,996] {dag.py:2287} INFO - Setting next_dagrun for dagrundag1 to None
[2024-02-27 09:23:11,009] {processor.py:170} INFO - Processing /opt/airflow/dags/dagrundag1.py took 0.065 seconds
[2024-02-27 09:23:16,634] {processor.py:162} INFO - Started process (PID=5164) to work on /opt/airflow/dags/dagrundag1.py
[2024-02-27 09:23:16,638] {processor.py:613} INFO - Processing file /opt/airflow/dags/dagrundag1.py for tasks to queue
[2024-02-27 09:23:16,640] {logging_mixin.py:109} INFO - [2024-02-27 09:23:16,639] {dagbag.py:496} INFO - Filling up the DagBag from /opt/airflow/dags/dagrundag1.py
[2024-02-27 09:23:16,657] {processor.py:623} INFO - DAG(s) dict_keys(['dagrundag1']) retrieved from /opt/airflow/dags/dagrundag1.py
[2024-02-27 09:23:16,709] {logging_mixin.py:109} INFO - [2024-02-27 09:23:16,709] {dag.py:1834} INFO - Sync 1 DAGs
[2024-02-27 09:23:16,748] {logging_mixin.py:109} INFO - [2024-02-27 09:23:16,748] {dag.py:2287} INFO - Setting next_dagrun for dagrundag1 to None
[2024-02-27 09:23:16,763] {processor.py:170} INFO - Processing /opt/airflow/dags/dagrundag1.py took 0.132 seconds
[2024-02-27 09:23:22,205] {processor.py:162} INFO - Started process (PID=5175) to work on /opt/airflow/dags/dagrundag1.py
[2024-02-27 09:23:22,206] {processor.py:613} INFO - Processing file /opt/airflow/dags/dagrundag1.py for tasks to queue
[2024-02-27 09:23:22,207] {logging_mixin.py:109} INFO - [2024-02-27 09:23:22,207] {dagbag.py:496} INFO - Filling up the DagBag from /opt/airflow/dags/dagrundag1.py
[2024-02-27 09:23:22,219] {processor.py:623} INFO - DAG(s) dict_keys(['dagrundag1']) retrieved from /opt/airflow/dags/dagrundag1.py
[2024-02-27 09:23:22,239] {logging_mixin.py:109} INFO - [2024-02-27 09:23:22,239] {dag.py:1834} INFO - Sync 1 DAGs
[2024-02-27 09:23:22,260] {logging_mixin.py:109} INFO - [2024-02-27 09:23:22,260] {dag.py:2287} INFO - Setting next_dagrun for dagrundag1 to None
[2024-02-27 09:23:22,274] {processor.py:170} INFO - Processing /opt/airflow/dags/dagrundag1.py took 0.073 seconds
[2024-02-27 09:23:27,943] {processor.py:162} INFO - Started process (PID=5186) to work on /opt/airflow/dags/dagrundag1.py
[2024-02-27 09:23:27,945] {processor.py:613} INFO - Processing file /opt/airflow/dags/dagrundag1.py for tasks to queue
[2024-02-27 09:23:27,947] {logging_mixin.py:109} INFO - [2024-02-27 09:23:27,946] {dagbag.py:496} INFO - Filling up the DagBag from /opt/airflow/dags/dagrundag1.py
[2024-02-27 09:23:27,959] {processor.py:623} INFO - DAG(s) dict_keys(['dagrundag1']) retrieved from /opt/airflow/dags/dagrundag1.py
[2024-02-27 09:23:27,986] {logging_mixin.py:109} INFO - [2024-02-27 09:23:27,986] {dag.py:1834} INFO - Sync 1 DAGs
[2024-02-27 09:23:28,000] {logging_mixin.py:109} INFO - [2024-02-27 09:23:28,000] {dag.py:2287} INFO - Setting next_dagrun for dagrundag1 to None
[2024-02-27 09:23:28,012] {processor.py:170} INFO - Processing /opt/airflow/dags/dagrundag1.py took 0.073 seconds
[2024-02-27 09:23:33,618] {processor.py:162} INFO - Started process (PID=5197) to work on /opt/airflow/dags/dagrundag1.py
[2024-02-27 09:23:33,625] {processor.py:613} INFO - Processing file /opt/airflow/dags/dagrundag1.py for tasks to queue
[2024-02-27 09:23:33,627] {logging_mixin.py:109} INFO - [2024-02-27 09:23:33,626] {dagbag.py:496} INFO - Filling up the DagBag from /opt/airflow/dags/dagrundag1.py
[2024-02-27 09:23:33,645] {processor.py:623} INFO - DAG(s) dict_keys(['dagrundag1']) retrieved from /opt/airflow/dags/dagrundag1.py
[2024-02-27 09:23:33,693] {logging_mixin.py:109} INFO - [2024-02-27 09:23:33,693] {dag.py:1834} INFO - Sync 1 DAGs
[2024-02-27 09:23:33,729] {logging_mixin.py:109} INFO - [2024-02-27 09:23:33,729] {dag.py:2287} INFO - Setting next_dagrun for dagrundag1 to None
[2024-02-27 09:23:33,766] {processor.py:170} INFO - Processing /opt/airflow/dags/dagrundag1.py took 0.151 seconds
[2024-02-27 09:23:39,235] {processor.py:162} INFO - Started process (PID=5208) to work on /opt/airflow/dags/dagrundag1.py
[2024-02-27 09:23:39,237] {processor.py:613} INFO - Processing file /opt/airflow/dags/dagrundag1.py for tasks to queue
[2024-02-27 09:23:39,240] {logging_mixin.py:109} INFO - [2024-02-27 09:23:39,240] {dagbag.py:496} INFO - Filling up the DagBag from /opt/airflow/dags/dagrundag1.py
[2024-02-27 09:23:39,268] {processor.py:623} INFO - DAG(s) dict_keys(['dagrundag1']) retrieved from /opt/airflow/dags/dagrundag1.py
[2024-02-27 09:23:39,305] {logging_mixin.py:109} INFO - [2024-02-27 09:23:39,305] {dag.py:1834} INFO - Sync 1 DAGs
[2024-02-27 09:23:39,327] {logging_mixin.py:109} INFO - [2024-02-27 09:23:39,327] {dag.py:2287} INFO - Setting next_dagrun for dagrundag1 to None
[2024-02-27 09:23:39,356] {processor.py:170} INFO - Processing /opt/airflow/dags/dagrundag1.py took 0.127 seconds
[2024-02-27 09:23:44,923] {processor.py:162} INFO - Started process (PID=5219) to work on /opt/airflow/dags/dagrundag1.py
[2024-02-27 09:23:44,924] {processor.py:613} INFO - Processing file /opt/airflow/dags/dagrundag1.py for tasks to queue
[2024-02-27 09:23:44,926] {logging_mixin.py:109} INFO - [2024-02-27 09:23:44,925] {dagbag.py:496} INFO - Filling up the DagBag from /opt/airflow/dags/dagrundag1.py
[2024-02-27 09:23:44,938] {processor.py:623} INFO - DAG(s) dict_keys(['dagrundag1']) retrieved from /opt/airflow/dags/dagrundag1.py
[2024-02-27 09:23:44,971] {logging_mixin.py:109} INFO - [2024-02-27 09:23:44,971] {dag.py:1834} INFO - Sync 1 DAGs
[2024-02-27 09:23:44,989] {logging_mixin.py:109} INFO - [2024-02-27 09:23:44,989] {dag.py:2287} INFO - Setting next_dagrun for dagrundag1 to None
[2024-02-27 09:23:45,005] {processor.py:170} INFO - Processing /opt/airflow/dags/dagrundag1.py took 0.085 seconds
[2024-02-27 09:23:50,633] {processor.py:162} INFO - Started process (PID=5230) to work on /opt/airflow/dags/dagrundag1.py
[2024-02-27 09:23:50,634] {processor.py:613} INFO - Processing file /opt/airflow/dags/dagrundag1.py for tasks to queue
[2024-02-27 09:23:50,635] {logging_mixin.py:109} INFO - [2024-02-27 09:23:50,635] {dagbag.py:496} INFO - Filling up the DagBag from /opt/airflow/dags/dagrundag1.py
[2024-02-27 09:23:50,646] {processor.py:623} INFO - DAG(s) dict_keys(['dagrundag1']) retrieved from /opt/airflow/dags/dagrundag1.py
[2024-02-27 09:23:50,675] {logging_mixin.py:109} INFO - [2024-02-27 09:23:50,675] {dag.py:1834} INFO - Sync 1 DAGs
[2024-02-27 09:23:50,690] {logging_mixin.py:109} INFO - [2024-02-27 09:23:50,690] {dag.py:2287} INFO - Setting next_dagrun for dagrundag1 to None
[2024-02-27 09:23:50,701] {processor.py:170} INFO - Processing /opt/airflow/dags/dagrundag1.py took 0.071 seconds
[2024-02-27 09:23:56,321] {processor.py:162} INFO - Started process (PID=5241) to work on /opt/airflow/dags/dagrundag1.py
[2024-02-27 09:23:56,323] {processor.py:613} INFO - Processing file /opt/airflow/dags/dagrundag1.py for tasks to queue
[2024-02-27 09:23:56,324] {logging_mixin.py:109} INFO - [2024-02-27 09:23:56,324] {dagbag.py:496} INFO - Filling up the DagBag from /opt/airflow/dags/dagrundag1.py
[2024-02-27 09:23:56,334] {processor.py:623} INFO - DAG(s) dict_keys(['dagrundag1']) retrieved from /opt/airflow/dags/dagrundag1.py
[2024-02-27 09:23:56,362] {logging_mixin.py:109} INFO - [2024-02-27 09:23:56,361] {dag.py:1834} INFO - Sync 1 DAGs
[2024-02-27 09:23:56,374] {logging_mixin.py:109} INFO - [2024-02-27 09:23:56,374] {dag.py:2287} INFO - Setting next_dagrun for dagrundag1 to None
[2024-02-27 09:23:56,388] {processor.py:170} INFO - Processing /opt/airflow/dags/dagrundag1.py took 0.071 seconds
[2024-02-27 09:24:01,878] {processor.py:162} INFO - Started process (PID=5252) to work on /opt/airflow/dags/dagrundag1.py
[2024-02-27 09:24:01,883] {processor.py:613} INFO - Processing file /opt/airflow/dags/dagrundag1.py for tasks to queue
[2024-02-27 09:24:01,884] {logging_mixin.py:109} INFO - [2024-02-27 09:24:01,884] {dagbag.py:496} INFO - Filling up the DagBag from /opt/airflow/dags/dagrundag1.py
[2024-02-27 09:24:01,895] {processor.py:623} INFO - DAG(s) dict_keys(['dagrundag1']) retrieved from /opt/airflow/dags/dagrundag1.py
[2024-02-27 09:24:01,923] {logging_mixin.py:109} INFO - [2024-02-27 09:24:01,923] {dag.py:1834} INFO - Sync 1 DAGs
[2024-02-27 09:24:01,936] {logging_mixin.py:109} INFO - [2024-02-27 09:24:01,936] {dag.py:2287} INFO - Setting next_dagrun for dagrundag1 to None
[2024-02-27 09:24:01,949] {processor.py:170} INFO - Processing /opt/airflow/dags/dagrundag1.py took 0.076 seconds
[2024-02-27 09:24:07,430] {processor.py:162} INFO - Started process (PID=5263) to work on /opt/airflow/dags/dagrundag1.py
[2024-02-27 09:24:07,431] {processor.py:613} INFO - Processing file /opt/airflow/dags/dagrundag1.py for tasks to queue
[2024-02-27 09:24:07,432] {logging_mixin.py:109} INFO - [2024-02-27 09:24:07,432] {dagbag.py:496} INFO - Filling up the DagBag from /opt/airflow/dags/dagrundag1.py
[2024-02-27 09:24:07,447] {processor.py:623} INFO - DAG(s) dict_keys(['dagrundag1']) retrieved from /opt/airflow/dags/dagrundag1.py
[2024-02-27 09:24:07,476] {logging_mixin.py:109} INFO - [2024-02-27 09:24:07,476] {dag.py:1834} INFO - Sync 1 DAGs
[2024-02-27 09:24:07,490] {logging_mixin.py:109} INFO - [2024-02-27 09:24:07,490] {dag.py:2287} INFO - Setting next_dagrun for dagrundag1 to None
[2024-02-27 09:24:07,502] {processor.py:170} INFO - Processing /opt/airflow/dags/dagrundag1.py took 0.076 seconds
[2024-02-27 09:24:13,039] {processor.py:162} INFO - Started process (PID=5274) to work on /opt/airflow/dags/dagrundag1.py
[2024-02-27 09:24:13,040] {processor.py:613} INFO - Processing file /opt/airflow/dags/dagrundag1.py for tasks to queue
[2024-02-27 09:24:13,041] {logging_mixin.py:109} INFO - [2024-02-27 09:24:13,041] {dagbag.py:496} INFO - Filling up the DagBag from /opt/airflow/dags/dagrundag1.py
[2024-02-27 09:24:13,052] {processor.py:623} INFO - DAG(s) dict_keys(['dagrundag1']) retrieved from /opt/airflow/dags/dagrundag1.py
[2024-02-27 09:24:13,080] {logging_mixin.py:109} INFO - [2024-02-27 09:24:13,079] {dag.py:1834} INFO - Sync 1 DAGs
[2024-02-27 09:24:13,094] {logging_mixin.py:109} INFO - [2024-02-27 09:24:13,094] {dag.py:2287} INFO - Setting next_dagrun for dagrundag1 to None
[2024-02-27 09:24:13,106] {processor.py:170} INFO - Processing /opt/airflow/dags/dagrundag1.py took 0.071 seconds
[2024-02-27 09:24:18,564] {processor.py:162} INFO - Started process (PID=5285) to work on /opt/airflow/dags/dagrundag1.py
[2024-02-27 09:24:18,570] {processor.py:613} INFO - Processing file /opt/airflow/dags/dagrundag1.py for tasks to queue
[2024-02-27 09:24:18,571] {logging_mixin.py:109} INFO - [2024-02-27 09:24:18,571] {dagbag.py:496} INFO - Filling up the DagBag from /opt/airflow/dags/dagrundag1.py
[2024-02-27 09:24:18,585] {processor.py:623} INFO - DAG(s) dict_keys(['dagrundag1']) retrieved from /opt/airflow/dags/dagrundag1.py
[2024-02-27 09:24:18,614] {logging_mixin.py:109} INFO - [2024-02-27 09:24:18,614] {dag.py:1834} INFO - Sync 1 DAGs
[2024-02-27 09:24:18,627] {logging_mixin.py:109} INFO - [2024-02-27 09:24:18,627] {dag.py:2287} INFO - Setting next_dagrun for dagrundag1 to None
[2024-02-27 09:24:18,640] {processor.py:170} INFO - Processing /opt/airflow/dags/dagrundag1.py took 0.079 seconds
[2024-02-27 09:24:24,141] {processor.py:162} INFO - Started process (PID=5296) to work on /opt/airflow/dags/dagrundag1.py
[2024-02-27 09:24:24,143] {processor.py:613} INFO - Processing file /opt/airflow/dags/dagrundag1.py for tasks to queue
[2024-02-27 09:24:24,144] {logging_mixin.py:109} INFO - [2024-02-27 09:24:24,144] {dagbag.py:496} INFO - Filling up the DagBag from /opt/airflow/dags/dagrundag1.py
[2024-02-27 09:24:24,154] {processor.py:623} INFO - DAG(s) dict_keys(['dagrundag1']) retrieved from /opt/airflow/dags/dagrundag1.py
[2024-02-27 09:24:24,183] {logging_mixin.py:109} INFO - [2024-02-27 09:24:24,183] {dag.py:1834} INFO - Sync 1 DAGs
[2024-02-27 09:24:24,198] {logging_mixin.py:109} INFO - [2024-02-27 09:24:24,198] {dag.py:2287} INFO - Setting next_dagrun for dagrundag1 to None
[2024-02-27 09:24:24,209] {processor.py:170} INFO - Processing /opt/airflow/dags/dagrundag1.py took 0.072 seconds
[2024-02-27 09:24:29,645] {processor.py:162} INFO - Started process (PID=5307) to work on /opt/airflow/dags/dagrundag1.py
[2024-02-27 09:24:29,647] {processor.py:613} INFO - Processing file /opt/airflow/dags/dagrundag1.py for tasks to queue
[2024-02-27 09:24:29,649] {logging_mixin.py:109} INFO - [2024-02-27 09:24:29,649] {dagbag.py:496} INFO - Filling up the DagBag from /opt/airflow/dags/dagrundag1.py
[2024-02-27 09:24:29,664] {processor.py:623} INFO - DAG(s) dict_keys(['dagrundag1']) retrieved from /opt/airflow/dags/dagrundag1.py
[2024-02-27 09:24:29,698] {logging_mixin.py:109} INFO - [2024-02-27 09:24:29,698] {dag.py:1834} INFO - Sync 1 DAGs
[2024-02-27 09:24:29,712] {logging_mixin.py:109} INFO - [2024-02-27 09:24:29,711] {dag.py:2287} INFO - Setting next_dagrun for dagrundag1 to None
[2024-02-27 09:24:29,726] {processor.py:170} INFO - Processing /opt/airflow/dags/dagrundag1.py took 0.086 seconds
